keras_imaging/preprocessing/standardize.py

```diff
@@ -73,3 +58,29 @@ def f(x):
         return y
 
     return f
+
+
+def normalization(means, max_values):
+    """
+    normalization make the image to range [-1,1]
+    :param means: means of image values across the dataset
+    :param max_values: maximum value
+    :return: normalized image
+    """
+    def f(x):
+        return (x - means) / max_values
```

Comment:
Maybe we should wrap [`skimage.exposure.rescale_intensity`](http://scikit-image.org/docs/dev/api/skimage.exposure.html#skimage.exposure.rescale_intensity)?